.clj-kondo/hooks/metabase/models/setting.clj

```diff
@@ -50,15 +59,15 @@
 (comment
   (defn- defsetting* [form]
     (hooks/sexpr
-     (:node
-      (defsetting
-        {:node
-         (hooks/parse-string
-          (with-out-str
-            (clojure.pprint/pprint
-             form)))}))))
+      (:node
+        (defsetting
+          {:node
+           (hooks/parse-string
+             (with-out-str
+               ((resolve 'clojure.pprint/pprint)
+                form)))}))))
 
-  (defn- x []
```

Comment:
Adding the usual kondo comment (^1) doesn't seem to work inside a `(comment ...)` block.

1. #_{:clj-kondo/ignore [:unused-private-value]}